runtime/lua/vim/lsp.lua

```diff
@@ -587,7 +587,7 @@ function lsp.enable(name, enable)
   else
     for _, nm in ipairs(names) do
       for _, client in ipairs(lsp.get_clients({ name = nm })) do
-        client:stop(client.exit_timeout)
+        client:stop()
```

Comment:
let's leave this explicit. that is clearer, and also avoids mistakes if we change our minds again.